app/ygnmi/cmd/generator/generator.go

```diff
@@ -45,41 +43,40 @@ func New() *cobra.Command {
 		Args:  cobra.MinimumNArgs(1),
 	}
 
-	generator.Flags().String("schema_struct_path", "", "The Go import path for the schema structs package. If struct generation is enabled, this defaults to base_import_path.")
+	generator.Flags().String("schema_struct_path", "", "The Go import path for the schema structs package. If struct generation is enabled, this defaults to base_package_path.")
 	generator.Flags().String("ygot_path", "github.com/openconfig/ygot/ygot", "The import path to use for ygot.")
 	generator.Flags().String("ygnmi_path", "github.com/openconfig/ygnmi/ygnmi", "The import path to use for ygnmi.")
 	generator.Flags().String("ytypes_path", "github.com/openconfig/ygot/ytypes", "The import path to use for ytypes.")
 	generator.Flags().String("goyang_path", "github.com/openconfig/goyang/pkg/yang", "The import path to use for goyang.")
-	generator.Flags().String("base_import_path", "", "This needs to be set to the import path of the output_dir.")
+	generator.Flags().String("base_package_path", "", "This needs to be set to the package path (module name + directories relative to go.mod file) of the output_dir.")
+	generator.Flags().String("trim_prefix", "", "A prefix (if any) to trim from generated package names and enums")
```

Comment:
I would prefer something like `trim_org_prefix` since we're specifically trimming the organization prefix from the YANG module names. This applies both for the go package names as well as the enum names.

Reply:
Or trim_module_prefix?

Reply:
i like trim_module_prefix